cmd/gke-exec-auth-plugin/request.go

```diff
@@ -22,12 +22,14 @@ import (
 )
 
 const (
-	kubeEnvMetadata = "instance/attributes/kube-env"
-	kubeEnvCert     = "TPM_BOOTSTRAP_CERT: "
-	kubeEnvKey      = "TPM_BOOTSTRAP_KEY: "
-	kubeEnvMaster   = "KUBERNETES_MASTER_NAME: "
-
-	caFilePath = "/etc/srv/kubernetes/pki/ca-certificates.crt"
+	kubeEnvMetadata   = "instance/attributes/kube-env"
+	kubeEnvCert       = "TPM_BOOTSTRAP_CERT: "
+	kubeEnvKey        = "TPM_BOOTSTRAP_KEY: "
+	kubeEnvMaster     = "KUBERNETES_MASTER_NAME: "
+	kubeEnvCaFilePath = "CA_FILE_PATH: "
```

Comment:
`kubeEnvCAFilePath` https://github.com/golang/go/wiki/CodeReviewComments#initialisms
same for `linuxCAFilePath`